code/modules/client/client_procs.dm

```diff
@@ -1175,3 +1175,16 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 
 /client/proc/increase_score(achievement_type, mob/user, value)
 	return player_details.achievements.increase_score(achievement_type, user, value)
+/// Attempts to make the client orbit the given object, for administrative purposes.
+/// If they are not an observer, will try to aghost them.
+/client/proc/admin_follow(atom/movable/target)
```

Comment:
we already have this proc exactly. It's in `code/modules/admin/admin_follow.dm`

Reply:
port artifact